documentation/docs/contributing/development-guide/setup.md

```diff
@@ -1,19 +1,41 @@
 ---
-title: 2 Initial setup
-sidebar_position: 2
+sidebar_position: 1
 ---
 
-Initial setup
-=============
+# Setup
 
-## Install pre-commit
+## Prerequisites
 
-```
+To work with the template monorepo you'll need to install [these packages](../../running/prerequisites.md).
+
+## Pre-commit
+
+When contributing to this project, pre-commit is necessary, as it runs certain tests, sanitisers, and formatters.
+
+The project provides a `.pre-commit-config.yaml` file that is used to setup git _pre-commit hooks_.
+
+On commit locally, code is automatically formatted, checked for security vulnerabilities using pre-commit git hooks.
+
+
+### Installation
+
+If you don't have it installed, run
+
+```shell
 pre-commit install
-# to enforce conventional commits
-pre-commit install --hook-type commit-msg
 ```
```

Comment:
Technically this descibes installing the pre-commit hooks we've created

Reply:
fixed